integration/bug-report-test.ts

```diff
@@ -47,30 +47,37 @@ test.beforeAll(async () => {
     // `createFixture` will make an app and run your tests against it.
     ////////////////////////////////////////////////////////////////////////////
     files: {
+      // XXX Remove .server from file for missing polyfill method error
+      "app/api/client.server.js": js`
+        import { AccountInfo } from '@azure/msal-node';
```

Comment:
This lib is not meant to be utilized in the browser and is expected to require / use a bunch of node polyfills.

Reply:
The issue I faced here was with using from a loader in a route which perhaps is considered client code but that wasn't clear to me.  I did solve it as per my previous comment by renaming the included file with `.server.ts` to it wasn't bundled but left this here due to the issue with the polyfills that looks to be closed now so is likely no longer an issue.